kuiper-game/Systems/Events/SetCourseEvent.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Linq;
+using Kuiper.Services;
+
+namespace Kuiper.Systems.Events
+{
+    public class SetCourseEvent : IEvent
+    {
+        IShipService _shipService;
+        public SetCourseEvent(IShipService shipService)
```

Comment:
DI Doesn't work here though, so i have to find a way to either make it work, or give it the shipService when it's created. Note that some events might need different services injected. A captain related event might need the captain service and a whole nother, but need both Ship and Captian service.